WarmUp5.java

```diff
@@ -0,0 +1,47 @@
+package com.exaple.warmup5;
+import java.util.Scanner;
+public class WarmUp5 {

```

Comment:
What you have is good (it works). Let make is OO thought. Refactor this code such that you have a class called Calculator.java which contains a hashtable of  Operator.java  which is a superclass of four operations (mul, add, div, subtract).